content/tutorial-reliable-transaction-submission.md

```diff
@@ -104,23 +105,49 @@ On restart, or the determination of a new last validated ledger (pseudocode):
 For each persisted transaction without validated result:
     Query transaction by hash
     If (result appears in validated ledger)
-		Persist the final result
-		If (result code is tesSUCCESS)
-			Application may act based on successful transaction
-		Else
-			Application may act based on failure
-			Maybe resubmit with new LastLedgerSequence and Fee
-	Else if (LastLedgerSequence > newest validated ledger)
-        Wait for more ledgers to be validated
-	Else
-		If (server has contiguous ledger history up to and
-	        including the ledger identified by LastLedgerSequence)
-			The transaction failed
-			Submit a new transaction, if appropriate for application
-		Else
-			Repeat submission of original transaction
+      # Outcome is final
```

Comment:
Consistently using 4 spaces for indentation and/or using {curly braces} might make this more readable.